deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,364 @@
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "prometheus/cluster"
+                  }
+                },
+                "name": "prometheus-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "prometheus/operator"
+                  }
+                },
+                "name": "prometheus-operator"
+              },
+              {
+                "kustomizeConfig": {
+                  "parameters": [
+                    {
+                      "name": "s3_endpoint_url",
+                      "value": "s3.odh.com"
+                    }
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "jupyterhub/jupyterhub"
+                  }
+                },
+                "name": "jupyterhub"
+              },
+              {
+                "kustomizeConfig": {
+                  "overlays": [
+                    "additional"
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "jupyterhub/notebook-images"
+                  }
+                },
+                "name": "notebook-images"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "airflow/operator"
+                  }
+                },
+                "name": "airflow-operator"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "airflow/cluster"
+                  }
+                },
+                "name": "airflow-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odhargo/cluster"
+                  }
+                },
+                "name": "odhargo-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odhargo/odhargo"
+                  }
+                },
+                "name": "odhargo"
+              }
+            ],
+            "repos": [
+              {
+                "name": "kf-manifests",
+                "uri": "https://github.com/opendatahub-io/manifests/tarball/v0.7-branch-openshift"
+              },
+              {
+                "name": "manifests",
+                "uri": "https://github.com/opendatahub-io/odh-manifests/tarball/master"
+              }
+            ],
+            "version": "v0.7-branch-openshift"
+          },
```

Comment:
Change to match the manifest version 0.6.1

Reply:
Updated